Casamari/MNC008.xml

```diff
@@ -408,28 +411,39 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <item xml:id="a2">
                            <desc type="GuestText"/>
                            <title ref="LIT2835RepCh119">Excerpt from the <foreign xml:lang="gez"
-                                 >Malkǝʾa Mikāʾel</foreign></title>
+                                 >Malkǝʾa <persName ref="PRS7049Michael"
+                                 >Mikāʾel</persName></foreign></title>
                            <desc>The excerpt comprises one complete stanza of the hymn.</desc>
                            <locus from="2r"/>
                            <q xml:lang="gez">
                               <seg>ሰላም፡ ለመታክፋቲከ፡ ዘረሰየ፡ ክዳኖ። ኤጲሞሰ፡ ዋካ፡ አራዘ፡ ተክህኖ። ሚከኤል፡ ፍንው፡ ለረድኤት፡
-                                 ወለአድኅኖ። ከመ፡ ሐረሳዊ፡ ዘይቀኒ፡ ተያፍኖ።መታክፈ፡ ፀርየ፡ ቅቅኒ፡ ወቅፃዕ፡ ዘባኖ።</seg>
+                                 ወለአድኅኖ። ከመ፡ ሐረሳዊ፡ ዘይቀኒ፡ ተያፍኖ። መታክፈ፡ ፀርየ፡ ቅቅኒ፡ ወቅፃዕ፡ ዘባኖ።</seg>
                            </q>
                         </item>
 
                         <item xml:id="a3">
                            <desc type="Unclear">List of individuals</desc>
                            <locus from="1v" to="2r"/>
                            <desc>The list is crudely written in pencil and includes the following
-                              names: <roleName type="title">ʾAbbā</roleName> Walda ʾIyasus,
-                                 <roleName type="title">ʾAbbā</roleName> Gabra Wald, <roleName
-                                 type="title">Qes</roleName> Gabra Wald, Tasfā Bǝyy, <roleName
-                                 type="title">Qes</roleName> Dastā, <roleName type="title"
-                                 >Qes</roleName> Warqǝnǝh, <roleName type="title">Qes</roleName>
-                              Walda Yoḥannǝs, <roleName type="title">Qes</roleName> Śāhle Kidāna
-                              Māryām, Gabra Yoḥannǝs, Gabra Wald. The list begins and ends with
-                              unclear scribbles. The name of Bayyanačč Zāwde is written with a blue
-                              ballpoint pen on <locus target="#2v"/>.</desc>
+                              names: <persName ref="PRS13340WaldaIya"><roleName type="title"
+                                    >ʾAbbā</roleName>Walda ʾIyasus</persName>, <persName
```

Comment:
Here and in the following names, a space should be left after `ʾAbbā</roleName>` or `Qes</roleName>` and before the proper name.

Reply:
I hate those spaces :)